x-pack/test/api_integration/apis/ml/saved_objects/initialize_trained_models.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { sortBy } from 'lodash';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+import { PutTrainedModelConfig } from '../../../../../plugins/ml/common/types/trained_models';
+
+type ModelType = 'regression' | 'classification';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const spacesService = getService('spaces');
+  const supertest = getService('supertestWithoutAuth');
+
+  const dfaJobId1 = 'ihp_od_1';
+  const dfaJobIdStarSpace = 'ihp_od_2';
+  const idSpace1 = 'space1';
+  const idSpace2 = 'space2';
+  const modelIdSpace1 = 'dfa1_classification_model_n_0';
+  const modelIdSpace2 = 'dfa2_classification_model_n_0';
+  const modelIdStarSpace = 'dfa3_classification_model_n_0';
+
+  async function runRequest(user: USER, expectedStatusCode: number) {
+    const { body, status } = await supertest
+      .get(`/api/ml/saved_objects/initialize`)
+      .auth(user, ml.securityCommon.getPasswordForUser(user))
+      .set(COMMON_REQUEST_HEADERS);
```

Comment:
should it be a part of `ml.api` methods?

Reply:
This is pattern of having a `runRequest` function is how most tests have been written and i've been corrected in the past for using `ml.api` functions for anything other than setup and validation checks.
@pheyos would have a better idea about this.